src/protocol/vm/state.rs

```diff
@@ -385,11 +401,11 @@ impl VMPoolState<PreCachedDB> {
                 .contains(&Capability::ScaledPrice)
             {
                 *price_result.first().ok_or_else(|| {
-                    TychoSimulationError::DecodingError("Expected a u64".to_string())
+                    TychoSimulationError::from(VMError::DecodingError("Expected a u64".to_string()))
```

Comment:
Here too. We need more details to help someone figure out why this error is occuring. What was meant to be a u64?